### PR TITLE
[Docs] Replace outdated content with refreshed content and links to docsite

### PR DESCRIPTION
**Pull request by jillnogold**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,662 +12,24 @@
 MLRun is an open-source MLOps framework that offers an integrative approach to managing your machine-learning pipelines from early development through model development to full pipeline deployment in production.
 MLRun offers a convenient abstraction layer to a wide variety of technology stacks while empowering data engineers and data scientists to define the feature and models.
```
Comment (yaronha) [inline README.md:14]: @jillnogold i suggest to add the architecture diagram and links to Katacoda and mlrun.org, and can embed the tutorial video (https://www.youtube.com/watch?v=O6g1pJJ609U)

Comment (jillnogold) [reply README.md:14]: done

File: README.md [hunk 2]
```diff
@@ -7,667 +7,73 @@
...
-from mlrun import get_or_create_ctx
-from mlrun.artifacts import ChartArtifact
-import pandas as pd
-
-
-def my_job(context, p1=1, p2='x'):
-    # load MLRUN runtime context (will be set by the runtime framework e.g. KubeFlow)
-
-    # get parameters from the runtime context (or use defaults)
+MLRun uses Serverless Function technology: write the code once, using your preferred development environment and simple “local” semantics, and then run it as-is on different platforms and at scale. MLRun automates the build process, execution, data movement, scaling, versioning, parameterization, outputs tracking, CI/CD integration, deployment to production, monitoring, and more.
```
Comment (yaronha) [inline README.md:16]: make `Serverless Function` bold

File: README.md [hunk 3]
```diff
@@ -7,667 +7,73 @@
...
 
-    # access input metadata, values, files, and secrets (passwords)
-    print(f'Run: {context.name} (uid={context.uid})')
-    print(f'Params: p1={p1}, p2={p2}')
-    print('accesskey = {}'.format(context.get_secret('ACCESS_KEY')))
-    print('file\n{}\n'.format(context.get_input('infile.txt', 'infile.txt').get()))
-    
-    # Run some useful code e.g. ML training, data prep, etc.
+Those easily developed data or ML “functions” can then be published or loaded from a marketplace and used later to form offline or real-time production pipelines with minimal engineering efforts.
```
Comment (yaronha) [inline README.md:18]: loaded from a [marketplace]..  add hyperlink to mlrun marketplace (https://www.mlrun.org/marketplace/)

File: README.md [hunk 4]
```diff
@@ -7,667 +7,73 @@
...
+- **[ML CI/CD pipeline](.docs//projects/ci-integration)** &mdash; automatically trains, tests, optimizes, and deploys or updates models using a snapshot of the production 
+data (generated by the feature store) and code from the source control (Git).
+- **[Real-Time Serving Pipeline](./docs/serving/serving-graph)** &mdash; Rapid deployment of scalable data and ML pipelines using real-time serverless technology, including 
+the API handling, data preparation/enrichment, [model serving](https://docs.mlrun.org/en/latest/serving/build-graph-model-serving.html), ensembles, driving and measuring actions, etc.
+- **[Real-Time monitoring and retraining](.docs//model_monitoring/index)** &mdash; monitors data, models, and production components and provides a feedback loop for exploring production data, identifying drift, alerting on anomalies or data quality issues, triggering re-training jobs, measuring business impact, etc.
+
+## Get started
+
+It's easy to start using MLRun: 
+1. Install MLRun using one of [over Kubernetes Cluster](https://docs.mlrun.org/en/latest/install/kubernetes.html) or [locally using Docker](https://docs.mlrun.org/en/latest/install/local-docker.html).<br>
```
Comment (yaronha) [inline README.md:43]: Install MLRun [locally using Docker] or [over Kubernetes Cluster]. Alternatively, you can use [Iguazio's managed MLRun service]

File: README.md [hunk 5]
```diff
@@ -7,667 +7,73 @@
...
+   Alternatively, you can [Use the managed MLRun service](https://www.iguazio.com/docs/latest-release/).
+2. [Set up your client environment](https://docs.mlrun.org/en/latest/install/remote.html) to work with the service. 
+3. Use the [Quick Start tutorial](https://docs.mlrun.org/en/latest/quick-start/quick-start.html) to develop and deploy machine learning applications to production.<br>
```
Comment (yaronha) [inline README.md:46]: Follow the [Quick Start tutorial] and [Additional Tutorials and Examples] to learn how to use MLRun to develop and deploy machine learning applications to production.

File: README.md [hunk 6]
```diff
@@ -7,667 +7,73 @@
...
+For hands-on learning, try the [MLRun Katakoda Scenarios](https://www.katacoda.com/mlrun). And you can watch the [Tutorial on Youtube](https://www.youtube.com/embed/O6g1pJJ609U) to see the flow in action.
```
Comment (yaronha) [inline README.md:47]: add new line before this

**Review by Hedingber — APPROVED** [review]

**Pull request merged**